client/battle/BattleWindow.cpp

```diff
@@ -733,6 +749,32 @@ std::optional<uint32_t> BattleWindow::getQueueHoveredUnitId()
 	return queue->getHoveredUnitIdIfAny();
 }
 
+void BattleWindow::endWithAutocombat() 
+{
+	if(!owner.makingTurn() || owner.tacticsMode)
+		return;
```

Comment:
Why not allow in tactics? With instant end of tactics phase?

Reply:
Didn't want to change that. Normal autobattle also excludes tactic mode.